packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -31,7 +32,7 @@ export class DataObjectWithCounter extends DataObject {
         this.counter = await handle.get();
     }
 
-    public async sendOp() {
+    protected sendOp() {
         assert(this.counter !== undefined, "Can't send ops when the counter isn't initialized!");
```

Comment:
You should make counter a getter and have this validation there. That will ensure no one can access counter without it being initilialized which is what you want.

Reply:
This assert is a handle assert, we want to store the counter in memory here